crates/steel-core/Cargo.toml

```diff
@@ -68,6 +68,8 @@ anyhow = { version = "1", optional = true }
 
 stacker = { version = "0.1.15", optional = true }
 
+bigdecimal = "0.4.5"
```

Comment:
Is this additional dependency needed?

Reply:
I used bigdecimals to cover cases with float and something possibly bigger than f64, like BigRational

Reply:
I'm not explicitly against bringing in another dependency, but what was the behavior before? Were we just losing precision then? I'm not sure if scheme even has big floats as a data type in general so I'm not explicitly sure if this is necessary

Reply:
Yes, precision, and need to manually handle Big* types conversion if they can't fit in f64 if we do without BigDecimal, to shortcut to "greater than"

Reply:
Got it, I think this is fine then. Sorry for the delay once again

Reply:
No worries, thanks!